src/android/patches/0.20/makeImportPatch.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function makeImportPatch(packageImportPath) {
+  return {
+    pattern: 'import com.facebook.react.ReactActivity;',
+    patch: '\n' + packageImportPath,
+  };
```

Comment:
On Android, we have an interface for the patch object: pattern + patch (both strings)